tests/braille-specs/en-us-mathtext.yaml

```diff
@@ -0,0 +1,144 @@
+# This yaml file tests the Braille display of the math symbols in the
+# en-us-mathtext.ctb table and the nemethdefs.cti subtable.
+# Copyright (C) 2023 Attila Hammer from IT Foundation for the Visually Impaired - Hungary. www.infoalap.hu
```

Comment:
Could you mention the documentation that was used to create these tests?

The problem with testing en-us-mathtext.ctb is that we don't really know what this table is used for and how it should behave. The table has got no table metadata either. So I don't know yet what to do with this.

An alternative approach could be to test nemeth.ctb, whenever we have moved it back to Liblouis (https://github.com/liblouis/liblouis/issues/815). But for this we also need to move the MathML tests to Liblouis, because we need a contract for the nemeth.ctb table. (The idea is explained in https://github.com/liblouis/liblouisutdml/pull/87.)

Reply:
> The problem with testing en-us-mathtext.ctb is that we don't really know what this table is used for and how it should behave. The table has got no table metadata either.

This really bugged me. I changed the test so that it does not test en-us-mathtext.ctb anymore. And I added the follow note:

> FIXME: This test should be changed so that it tests a top-level table (i.e. a table that can be discovered based on metadata). For instance, the table could be nemeth.ctb, and the test would document the math symbols as well as the "semantic" characters used for the linearization of MathML (see also https://github.com/liblouis/liblouis/issues/815).